### PR TITLE
Seeding

### PR DESCRIPTION
**Pull request by ryuuohlee**
Got seeding working and servers connected.

### CODE REVIEW

File: database/connect.js [hunk 1]
```diff
@@ -18,4 +20,5 @@ connection.connect(function(err){
   console.log('Ikea details Connected');
 });
 
+
```
Comment (ingdjason) [inline database/connect.js]: where i can see the config setting for dB connection

Comment (ryuuohlee) [reply database/connect.js]: It should be in that file. It is in the first 15 lines.

Comment (ingdjason) [reply database/connect.js]: sorry, i did'nt click on the button .

Comment (ingdjason) [reply database/connect.js]: Perfect

**Pull request merged**